### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by hannaholl**
Add basic instructions on how to run keeper.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+createdb keeper-dev
+```
+
+If creating the db gives you an error, postgresql might not be running. To start it run this:
+```
+brew services start postgresql
+```
+
+### 3. Update the keeper config file
+In `server/server/config/config.json`, change the username `dan` to the username on your machine. If you don't know what it is, `whoami` will give it to you.
```
Comment (hannaholl) [inline README.md]: The username can probably be an environment variable later.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# keeper
+
+## Setup
+
+This is how to get keeper running on your machine so that you're able to work on it.
+
+First you have to login to clubhouse and generate an API key. You can do this from your account settings -> API key. Save this token somewhere and add it to a new environment variable called `CLUBHOUSE_API_KEY`.
```
Comment (hannaholl) [inline README.md]: We should probably add more details on how to do this but for now I think it's better than nothing at least.

Comment (danbahrami) [inline README.md]: could specify that environment variables should be added to `/server/.env`

File: README.md (hunk 1 above)
Comment (danbahrami) [reply README.md]: yeah good point 🤔

File: README.md (hunk 2 above)
Comment (hannaholl) [reply README.md]: So I haven't actually done this and don't know how it works? Do you want to add it or explain it and I can write it to the readme?

Comment (danbahrami) [reply README.md]: So you can create a new file called `.env` in the `server` directory and add the line

```
CLUBHOUSE_API_KEY=abcd-efgh-ijkl
```

**Pull request merged**